code42_connector.py

```diff
@@ -65,19 +74,47 @@ def _handle_test_connectivity(self, param):
             # Ideally the client instantiation would happen in `initialize()` but that function does not have access
             # to the action, so it cannot effectively report errors to the UI.
             # Fix this with a decorator or something similar.
-            self._client = py42.sdk.from_local_account(self._cloud_instance, self._username, self._password)
-            self._client.users.get_current()
+            self.client.users.get_current()
         except Exception as exception:
             return action_result.set_status(phantom.APP_ERROR, f"Unable to connect to Code42: {str(exception)}")
 
         self.save_progress("Test Connectivity Passed")
         return action_result.set_status(phantom.APP_SUCCESS)
 
+    def _handle_add_departing_employee(self, param):
+        self.save_progress(f"In action handler for: {self.get_action_identifier()}")
+        action_result = self.add_action_result(ActionResult(dict(param)))
+        username = param["username"]
+        departure_date = param.get("departure_date")
+        user_id = self._get_user_id(username)
+        response = self.client.detectionlists.departing_employee.add(user_id, departure_date=departure_date)
+        action_result.add_data(response.data)
```

Comment:
what does `response.data` hold for these APIs? Is it worth including?

Reply:
See the tests!

There is nothing in the remove call, but the add call adds stuff.. Adding the response by default here doesn't really hurt and seems to be the norm. I think we may still have to define outputs in the json file in order for them to work in playbooks.

Reply:
So, if like XSOAR, there are 3 groups:

1.) The raw full response data
2.) The table data
3.) Action outputs (for playbook action chaining)